scripts/mkcloud

```diff
@@ -140,6 +140,7 @@ function show_environment
     end_time=`date`
     echo "Environment Details"
     echo "-------------------------------"
+    echo "         git: $(git --no-pager -C $SCRIPTS_DIR show -s --oneline)"
```

Comment:
Is it really safe to assume that `git` is always installed?  E.g. we have partners using `mkcloud` now, and they might be downloading a tarball from github.  In any case I'd prefer some more defensive programming here, e.g. it could only output this line if `which git >&/dev/null` exits 0.